File: app/src/main/java/com/medialink/mysqlite/DatabaseContract.java
package com.medialink.mysqlite;

import android.provider.BaseColumns;

public class DatabaseContract {
    public static String TABLE_NOTE = "TABLE_NOTE";

    public static final class NoteColumns implements BaseColumns {
        public static String TITLE = "TITLE";
        public static String DESCRIPTION = "DESCRIPTION";
        public static String DATE = "DATE";
    }
}

File: MySQLiteKotlin/src/main/java/com/medialink/mysqlitekotlin/db/DatabaseContract.kt
package com.medialink.mysqlitekotlin.db

import android.provider.BaseColumns

class DatabaseContract {
    class NoteColumns : BaseColumns {
        companion object {
            const val TABLE_NOTE = "TABLE_NOTE"
            const val _ID ="_id"
            const val TITLE = "TITLE"
            const val DESCRIPTION = "DESCRIPTION"
            const val DATE = "DATE"
        }
    }
}